partcad/src/partcad/context.py

```
@@ -71,7 +71,7 @@ def __enter__(self, *_args):
         def __exit__(self, *_args):
             self.lock.release()
 
-    def __init__(self, root_path=None, search_root=True):
+    def __init__(self, root_path=None, search_root=True, user_config=UserConfig()):
```

Ruff (0.8.2)

Do not perform function call `UserConfig` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)